<launch>
    <node name="aruco_detector" pkg="aruco_perception" type="aruco_detector" output="screen">
        <param name="aruco_dictionary" value="DICT_6X6_250"/>
        <param name="display_output" value="true"/>
    </node>
</launch>
